Duanaire_Finn/PoemXI.trig

```diff
@@ -15,7 +15,7 @@
     a dctype:Dataset;
     dcterms:title "Poem XI";
     dcterms:isFormatOf <https://archive.org/stream/duanairefinnboo00fostgoog#page/n96/mode/2up>;
-    dcterms:format "text/turtle" ;
+    dcterms:format "text/trig" ;
```

Comment:
This needs to be "application/trig" in all instances.  It is a pain but for some reason they did not use the same MIME type for the format.

Reply:
Ok, it's now "application/trig". How does it look?